src/sagemaker/huggingface/estimator.py

```diff
@@ -141,6 +141,16 @@ def __init__(
                             }
                         }
                     }
+
+                To enable distributed training with Training Compiler for PyTorch:
+
+                .. code:: python
+
+                    {
+                        "pytorch_xla": {
```

Comment:
Let's make this `pytorchxla` to be consistent with:
`distribution = {"pytorchddp": {"enabled": "true"}}`